R/flatten.R

```diff
@@ -52,12 +52,29 @@
 #' Since `join = nest_join()` does not make sense in this direction (LHS = child table, RHS = parent table: for valid key constraints
 #' each nested column entry would be a tibble of one row), an error will be thrown if this method is chosen.
 #'
+#' The difference between `dm_flatten_to_tbl()` and `dm_squash_to_tbl()` is
+#' the following (see the examples):
+#'
+#' - `dm_flatten_to_tbl()` allows only one level of hierarchy
```

Comment:
Could there be more details here?

Also instead of a "Details" header could there be a section "Flatten vs squash"?

Lastly, could there be a brief description of intended use cases?